### PR TITLE
Go 1.10.1

### PR DESCRIPTION
**Pull request by ydnar**
Blocked on https://hub.docker.com/r/_/golang/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -64,10 +64,10 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true; \
 	apt-get update \
 	&& apt-get -q -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confnew" upgrade \
 	&& apt-get install -q -y --no-install-recommends \
-		apt-transport-https \
-		software-properties-common \
-		netcat-traditional netcat zip \
-		git-hub less \
```
Comment (cee-dub) [inline Dockerfile:70]: Why all the whitespace changes?

Comment (ydnar) [reply Dockerfile:70]: Looks like VS Code fixing a few space runs, and un-indenting lines it views as incorrect?

Comment (cee-dub) [reply Dockerfile:70]: These lines to me look like they should be as indented as they were.

File: Dockerfile [hunk 2]
```diff
@@ -81,16 +81,16 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true; \
 	&& curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | gpg --dearmor > /etc/apt/keys/docker.gpg \
 	&& printf > /etc/apt/preferences.d/docker.pref 'Package: *\nPin: origin download.docker.com\nPin-Priority: 100\n' \
 	&& echo \
-		"deb [arch=amd64 signed-by=/etc/apt/keys/docker.gpg] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")" \
-		$(lsb_release -cs) \
-		stable \
-		> /etc/apt/sources.list.d/docker.list \
```
Comment (cee-dub) [inline Dockerfile:87]: Same, these look like they should stay indented.

File: Dockerfile [hunk 3]
```diff
@@ -81,16 +81,16 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true; \
...
+	"deb [arch=amd64 signed-by=/etc/apt/keys/docker.gpg] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")" \
+	$(lsb_release -cs) \
+	stable \
+	> /etc/apt/sources.list.d/docker.list \
 	&& apt-get update \
 	&& apt-get -q -y install docker-ce \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN groupadd -g "${RUNTIME_GID}" "${RUNTIME_USER}" && \
-    useradd -p '*' -u "${RUNTIME_UID}" -g "${RUNTIME_USER}" -G "${RUNTIME_SUPGIDS}" -m "${RUNTIME_USER}"
```
Comment (cee-dub) [inline Dockerfile:93]: Likewise.

**Comment by philpennock** [comment]
Agree with Cameron re leaving the whitespace alone, but the change itself LGTM.

**Comment by philpennock** [comment]
NB: docker hub now has 1.10.1

**Comment by philpennock** [comment]
*CORRECTION:* the full description at https://hub.docker.com/r/_/golang/ claims to have `1.10.1` but the actual tags available -- at https://hub.docker.com/r/library/golang/tags/ -- do **NOT** include `1.10.1`, despite the last push being 3 days ago, so something has gone wrong on Docker's side.

**Comment by ydnar** [comment]
@philpennock this working now?

**Comment by philpennock** [comment]
Looks like it.

**Pull request closed without merging**